library/composer.json

```diff
@@ -6,6 +6,7 @@
         "h4cc/wkhtmltopdf-amd64": "0.12.3",
         "h4cc/wkhtmltopdf-i386": "0.12.3",
         "crada/php-apidoc":"@dev",
-        "xamin/handlebars.php": "^0.10.4"
+        "xamin/handlebars.php": "^0.10.4",
+        "debach/zend-mp3": "dev-master"
```

Comment:
1.8.1 it's fine. You could use 1.8.* if you prefer, or any other "patches only" contraint as described in https://getcomposer.org/doc/articles/versions.md#writing-version-constraints